src/components/PlayerCard/PlayerCard.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect, useState } from 'react';
+import { Props } from './types';
+
+export default function PlayerCard(props: Props) {
+  const [readyStatus, setReadyStatus] = useState(false);
+  useEffect(() => {
+    if (props.playerType === 'bot') {
```

Comment:
❗️
Типы надо выносить в `enum`. Подобное никогда не стоит юзать без референсов на определённое где-то значение, потому что в противном случае можно наловить неконсистентностей и больно их искать
```suggestion
    if (props.playerType === PlayerType.Bot) { // ну или как-то так
```